app/test/setup-test-framework.ts

```diff
@@ -0,0 +1,3 @@
+/* eslint-disable strict */
```

Comment:
😞

Reply:
Why can’t you add the `use strict` directive?

Reply:
shouldn't the typescript compiler be doing that for me?

Reply:
According to [this Stack Overflow answer](https://stackoverflow.com/a/31392947/5244995), it still has some benefits.